src/header/header.scss

```diff
@@ -116,6 +120,31 @@
       }
     }
   }
+
+  @media (max-width: 768px) {
+    &,
+    &.iui-slim.iui-slim {
+      .iui-left {
+        .iui-header-button {
+          .iui-header-button-icon.iui-header-button-icon {
+            @include iui-display-none;
+          }
+
+          .iui-header-button-icon + .iui-label {
+            margin-left: 0;
+          }
+        }
+      }
+    }
+  }
+
+  @media (max-width: 1024px) {
+    .iui-left {
+      .iui-header-logo > .iui-label {
+        @include iui-display-none;
```

Comment:
Is it really needed? As I understand it is only for animation. I don't think it should be a main case that users resize window so we could use `display: none`. Plus Jon's comment about search engines.

Reply:
Yeah but animations are nice 😄

Reply:
I thought replacing that mixin with `display: none` everywhere.

Reply:
Oh, idk about that. It is very much needed in button description (and not for resizing window).